soc/nordic/common/vpr/Kconfig.sysbuild

```
@@ -4,7 +4,7 @@
 config VPR_LAUNCHER
 	bool "VPR launcher"
 	default y
-	depends on (SOC_NRF54H20_CPUPPR || SOC_NRF54H20_CPUFLPR || SOC_NRF54L15_CPUFLPR || SOC_NRF9280_CPUPPR)
+	depends on (SOC_NRF54H20_CPUPPR ||  SOC_NRF54H20_CPUFLPR ||  SOC_NRF54L_05_10_15_CPUFLPR || SOC_NRF9280_CPUPPR)
```

Actually, this could be simplified to:
```suggestion
	depends on RISCV_CORE_NORDIC_VPR
```